source/riteway.js

```diff
@@ -24,9 +24,9 @@ const describe = (unit = '', TestFunction = noop) => tape(unit, test => {
   if (result && result.then) return result.then(end);
 });
 
-const Try = (fn = noop, ...args) => {
+const Try = async (fn = noop, ...args) => {
```

Comment:
I'm trying to image if making Try promise-aware would be a benefit (just to avoid forcing the use of `await` when not required, and keep this change non-breaking).

```js
const Try = (fn = noop, ...args) => {
  try {
    const x = fn(...args)
    if (typeof x === 'object' && o.constructor === Promise)
      return x.catch(identity)
  } catch (err) {
    return err
  }
}
```

```ts
export function Try<U extends any[], V>(fn: (...args: U) => V, ...args: U): any | Promise<any>
```

Reply:
```js
if (typeof x === 'object' && o.constructor === Promise)
```

This will break across execution contexts. Try ducktyping, instead:

```js
if (x && typeof x.then === 'function')
```